setup.py

```diff
@@ -35,6 +35,11 @@
     entry_points={
         'console_scripts': [
             'ivadomed=ivadomed.main:run_main',
```

Comment:
is `ivadomed` a callable script? what does it do?

Reply:
ivadomed config_file.json Calls the main.py script. So it's for training/evaluation with a config file

Reply:
I see. So maybe we should use `ivadomed_` suffix for consistency

Reply:
You mean instead of imed_?

Reply:
> You mean instead of imed_?

yes, that's what i meant

Reply:
with "tab" autocompletion, i suspect ppl will only have to write "iva+TAB" to get the list of functions

Reply:
Right ! Done in 4f6f3f8510